test/github-issues/184/issue-184.ts

```diff
@@ -26,7 +26,7 @@ describe("github issues > #184 [Postgres] Single-Inheritance not working with in
         employee.shared = "e";
 
         await employeeRepository.save(employee);
-        await employeeRepository.findOne(1);
+        await employeeRepository.findOne("1");
```

Comment:
Id is a string. In the past we passed ids always as parameters(conversion to string was done inside pg library), now we pass them in query content if they're numbers.